### PR TITLE
feat(frontend): substitute roadmap with releases in Menu

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We replace the Oisy Roadmap with the Changelog in the Menu.

### Before

Screenshot 2024-09-24 at 15 30 44

### After

Screenshot 2024-09-24 at 17 09 46

### CODE REVIEW

**Review by anedos-dfinity — APPROVED** [review]

**Review by DenysKarmazynDFINITY — APPROVED** [review]
LGTM, thanks!

**Comment by peterpeterparker** [comment]
Unsolicited feedback @AntonioVentilii-DFINITY : I would rename the menu point "Oisy Wallet Releases" into "Changelog" 

Shorter and more meaningful in my opinion.

**Comment by peterpeterparker** [comment]
Btw. on Juno the releases are read at build time and converted to static pages: https://juno.build/changelog
If that's something we want to build, it's possible too

File: src/frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -29,13 +29,13 @@
...
 			"source_code": "Source code",
-			"oisy_roadmap": "$oisy_name Roadmap",
+			"changelog": "Changelog",
 			"submit_ticket": "Submit a ticket"
 		},
 		"alt": {
 			"more_settings": "More settings",
 			"menu": "Your wallet address, settings, sign-out and external links",
-			"oisy_roadmap": "Open the $oisy_name roadmap",
+			"changelog": "Open the $oisy_name changelog",
```
Comment (peterpeterparker) [inline src/frontend/src/lib/i18n/en.json:38]: Nitpick: this alt can probably be more descriptive, e.g. "Open the changelog of $oisy_name on GitHub to review the latest updates"

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/i18n/en.json:38]: changed

**Pull request merged**